storage.go

```diff
@@ -49,6 +50,19 @@ func (s *Storage) copy(ctx context.Context, src string, dst string, opt pairStor
 	return nil
 }
 
+func (s *Storage) create(path string, opt pairStorageCreate) (o *Object) {
+	// FIXME: should set false or true here?
+	o = s.newObject(true)
```

Comment:
I think we should set it to `false`.

Becase the object created by user will always missing some attrs like `storage-class` and so on. Even user provides all the attr, we still need to send a `stat` to make sure.